src/icalendar/cal/component.py

```diff
@@ -71,14 +82,19 @@ def get_component_class(cls, name: str) -> type[Component]:
         return cls._components_factory.get(name, Component)
 
     @staticmethod
-    def _infer_value_type(value: date | datetime | timedelta | time | tuple | list) -> str | None:
+    def _infer_value_type(
+        value: date | datetime | timedelta | time | tuple | list,
```

Comment:
Why a trailing comma? Looks like an overly eager formatter pooped a comma in your code.

Please resist reformatting code that you otherwise wouldn't touch. It bloats PRs. A separate PR that only reformats the entire code base is easier to manage.